licenses/license_tinyfiledialogs.txt

```diff
@@ -0,0 +1,47 @@
+--------------------------------------------------------------------------------
```

Comment:
Would you mind adding this to the [licenses/README.md](https://github.com/QB64-Phoenix-Edition/QB64pe/blob/main/licenses/README.md) in the `QB64 Phoenix Edition Runtime` table?

Reply:
Done. Please check the lines added though.